### DISCUSSION TITLE
service user / access users from different organization

### DISCUSSION THREAD

**Discussion by aheissenberger** [zitadel/zitadel#7797] [Q&A] [answered]
How can I **grant a service user** from one organization **access** to users from an **other organization**.

**Use Case:** I need to add/change Meta Data of a user (from org "B") which logged into an application (project of org "A") which allows login from all organizations in the zitadel instance.

**Steps to prepare the service user:**
1. the service user "SU" was created in the organization "A" which owns the project with the app
2. the service user "SU"  was granted the role "Org Owner"
3. on organization "B" the service user "SU" was added as a manager from an external organization with the role "Org Owner"

The service user "SU" gets the user info from organization "A" for a user per user id if the user is a user of org. "A".
The service user "SU"  **fails to get the user info** for a user from **organization "B"**

Is there an option to create a service user with a reduced set of permission? e.g. only to change the meta data of a user and change/assign roles?

**Comment by hifabienne** [comment, marked as answer]
I assume you are using the management api.
Per default the request will always be performed in the organization of the requesting user if nothing else is specified.
with the x-zitadel-orgid header you can specify on which organization you like to perform the request

**Reply by aheissenberger** [reply]
Thanks - I just found out - that with the API `/v2beta/users/:Id` ist is possible without the header
